infrastructure/database.tf

```diff
@@ -25,19 +25,19 @@ module "postgresql" {
 resource "azurerm_key_vault_secret" "POSTGRES-USER" {
   name         = "civil-sdt-POSTGRES-USER"
   value        = module.postgresql.username
-  key_vault_id = module.civil_sdt_key_vault.key_vault_id
+  key_vault_id = data.azurerm_key_vault.key_vault.id
 }
 
 # Create secret for database password
 resource "azurerm_key_vault_secret" "POSTGRES-PASS" {
   name         = "civil-sdt-POSTGRES-PASS"
   value        = module.postgresql.password
-  key_vault_id = module.civil_sdt_key_vault.key_vault_id
+  key_vault_id = data.azurerm_key_vault.key_vault.id
```

Comment:
Change key_vault_id value to `module.civil_sdt_key_vault.key_vault_id` as secret needs to be created in the civil-sdt key vault.